docs/image-acquisition/network-cameras.md

```diff
@@ -0,0 +1,40 @@
+Just like [local webcams](image-acquisition/video-file.md), you can extract frames from IP cameras.
+
+The example below iterates over the first 100 frames captured by the network camera, resizes it and saves it to
+a new video file.
+
+```py
+from landingai.pipeline.image_source import NetworkedCamera
+from landingai.pipeline.frameset import FrameSet
+
+frameset = FrameSet() # (1)!
+with NetworkedCamera(stream_url="rtsp://192.168.0.77:8080/h264_opus.sdp", fps=1) as camera: # (2)!
+    for i, frame in enumerate(camera): # (3)!
+        if i >= 100:
+            break
+        frame.resize(width=256) # (4)!
```

Comment:
if previously you were applying frame.resize(width=512, height=512) to keep aspect ratio, why now you only resize to the width to 256?

Reply:
The previous comment was wrong. It was not keeping aspect ratio.